### PR TITLE
[Fix/EPMEDU-1481] Add app version to the page "About" footer 

### PR DESCRIPTION
**Pull request by ozkarram**
Adds App version footer on About page.

### CODE REVIEW

File: feature/tabsflow/moreflow/about/src/main/java/com/epmedu/animeal/tabs/more/about/AboutScreenUI.kt [hunk 1]
```diff
@@ -78,6 +81,13 @@ internal fun AboutScreenUI(
             )
         }
         Spacer(modifier = Modifier.weight(1f))
+        Text(
+            modifier = Modifier.fillMaxWidth(),
+            text = stringResource(id = R.string.page_about_version, currentVersion),
+            style = MaterialTheme.typography.body1,
+            color = Color(0xFF898A8D),
```
Comment (VEZE) [inline feature/tabsflow/moreflow/about/src/main/java/com/epmedu/animeal/tabs/more/about/AboutScreenUI.kt]: remove hardcode, use color palette

File: feature/tabsflow/moreflow/about/src/main/java/com/epmedu/animeal/tabs/more/about/AboutScreenState.kt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+package com.epmedu.animeal.tabs.more.about
+
+data class AboutScreenState(
+    val currentVersion: String = ""
```
Comment (VEZE) [inline feature/tabsflow/moreflow/about/src/main/java/com/epmedu/animeal/tabs/more/about/AboutScreenState.kt]: use const variable for emtpy strings

**Review by VEZE — CHANGES_REQUESTED** [review]

File: feature/tabsflow/moreflow/about/src/main/java/com/epmedu/animeal/tabs/more/about/AboutScreenViewModel.kt [hunk 3]
```diff
@@ -0,0 +1,20 @@
+package com.epmedu.animeal.tabs.more.about
```
Comment (Fileek) [inline feature/tabsflow/moreflow/about/src/main/java/com/epmedu/animeal/tabs/more/about/AboutScreenViewModel.kt]: Please create a sub-package for viewmodel `com.epmedu.animeal.tabs.more.about.viewmodel`

File: feature/tabsflow/moreflow/about/src/main/java/com/epmedu/animeal/tabs/more/about/AboutScreenState.kt [hunk 4]
```diff
@@ -0,0 +1,7 @@
+package com.epmedu.animeal.tabs.more.about
```
Comment (Fileek) [inline feature/tabsflow/moreflow/about/src/main/java/com/epmedu/animeal/tabs/more/about/AboutScreenState.kt]: State classes should also be in viewmodel sub-package

**Review by Fileek — CHANGES_REQUESTED** [review]

File: feature/tabsflow/moreflow/about/src/main/java/com/epmedu/animeal/tabs/more/about/AboutScreenState.kt (hunk 2 above)
Comment (ozkarram) [reply feature/tabsflow/moreflow/about/src/main/java/com/epmedu/animeal/tabs/more/about/AboutScreenState.kt]: Changes added

File: feature/tabsflow/moreflow/about/src/main/java/com/epmedu/animeal/tabs/more/about/AboutScreenState.kt (hunk 4 above)
Comment (ozkarram) [reply feature/tabsflow/moreflow/about/src/main/java/com/epmedu/animeal/tabs/more/about/AboutScreenState.kt]: changes added

**Review by Fileek — APPROVED** [review]

**Pull request merged**